plugins/services/src/js/events/SDKEndpointActions.js

```diff
@@ -0,0 +1,77 @@
+import { RequestUtil } from "mesosphere-shared-reactjs";
+
+import AppDispatcher from "#SRC/js/events/AppDispatcher";
+import {
+  REQUEST_SDK_ENDPOINTS_SUCCESS,
+  REQUEST_SDK_ENDPOINTS_ERROR,
+  REQUEST_SDK_ENDPOINT_SUCCESS,
+  REQUEST_SDK_ENDPOINT_ERROR,
+  REQUEST_SDK_ENDPOINTS_LOADING
+} from "../constants/ActionTypes";
+
+const SDKEndpointsActions = {
+  fetchEndpoints(serviceId) {
+    AppDispatcher.handleServerAction({
+      type: REQUEST_SDK_ENDPOINTS_LOADING,
```

Comment:
I don't think we should introduce this pattern unless there is a strong argument for introducing it. Please follow our existing pattern in react components for identifying loading state or discuss in advance the need for such an introduction, before implementing it :)

Reply:
Removed